package.json

```diff
@@ -60,7 +60,7 @@
     "npm": "^2.14.0"
   },
   "scripts": {
-    "build": "gulp",
+    "build": "git clone -b fix\/tracker-ebola git://github.com/frictionlessdata/jsontableschema-js.git node_modules/jsontableschema && cd node_modules/jsontableschema && npm install && npm run build && gulp",
```

Comment:
Yes, it will run build of jsontableschema.
May be just add jsontableschema builded scripts into the project for now? Heroku do only "npm install" and use "public" folder to run the app